WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -299,18 +317,19 @@ public void onClick(View v) {
     }
 
     private void initSmartLockForPasswords() {
-        if (!isGooglePlayServicesAvailable()) {
+        if (!isSmartLockAvailable()) {
             return;
         }
+
         mCredentialsClient = new GoogleApiClient.Builder(getActivity())
                 .addConnectionCallbacks(this)
-                .enableAutoManage((SignInActivity) getActivity(), this)
+                .enableAutoManage(getActivity(), this)
                 .addApi(Auth.CREDENTIALS_API)
                 .build();
     }
 
     public void smartLockAutoFill() {
-        if (!isGooglePlayServicesAvailable() || mCredentialsClient == null) {
+        if (!isSmartLockAvailable()) {
```

Comment:
Same here, we shouldn't use `isSmartLockAvailable` (this method should work when `mCredentialsClient` is disconnected)